src/App/pages/ReposSearchPage/ReposSearchPage.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+
+import Button from "@components/Button";
+import Input from "@components/Input";
+import RepoTile from "@components/RepoTile";
+import SearchIcon from "@components/SearchIcon";
+//import { useReposListContext } from "../../App";
+import ReposListStore from "@store/ReposListStore";
+import rootStore from "@store/RootStore";
+import { useQueryParamsStoreInit } from "@store/RootStore/hooks/useQueryParamsStoreInit";
+import { Meta } from "@utils/meta";
+import { useLocalStore } from "@utils/useLocalStore";
+import { toJS } from "mobx";
+import { observer } from "mobx-react-lite";
+
+import styles from "./ReposSearchPage.module.scss";
+
+const ReposSearchPage: React.FC = () => {
+    useQueryParamsStoreInit();
+
+    const reposListStore = useLocalStore(() => new ReposListStore());    
+    
+    //console.log('render', toJS(reposListStore.list));
+    React.useEffect(() => {
+        const search = rootStore.query.getParam("search")?.toString();
+
+        if (search){
+            reposListStore.getOrganizationReposList({
+                org: search,
+                query: {
+                  per_page: 5
+                }
+            });
+
+            console.log(reposListStore.list);
+            console.log(reposListStore.meta);
+        }        
+    }, [reposListStore]);
+
+    const handleOnChange = (event): void => {
```

Comment:
нужно обернуть в useCallback

```suggestion
    const handleOnChange = React.useCallback((value: string): void => {
```